src/toolbar.scss

```diff
@@ -0,0 +1,170 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-toolbar;
+$toolbar-height-compact: 2rem;
+$toolbar-height-cozy: 2.75rem;
+
+$toolbar-separator-height-compact: 1.5rem;
+$toolbar-separator-height-cozy: 2rem;
+
+$toolbar-spacing: 0.5rem;
+$toolbar-padding: 0 0.5rem;
+$title-toolbar-padding: 0 0.5rem 0 0.1rem;
+$title-toolbar-height: 2.75rem;
+
+%toolbarStandardColors {
+  @include fd-hover() {
+    background-color: var(--sapList_Hover_Background);
+  }
+
+  @include fd-active() {
+    background-color: var(--sapActiveColor);
+  }
+}
+
+@mixin activeStyles() {
+  &.#{$block}--active {
+    @content;
+  }
+}
+
+@mixin toolbarAuto() {
+  @include activeStyles() {
+    @extend %toolbarStandardColors;
```

Comment:
I believe the background color of Auto should be `inherit` since it's based on the parent

Reply:
fixed